forms.css

```
@@ -22,11 +22,18 @@ textarea {
 select {
   -moz-appearance: none;
   -webkit-appearance: none;
-  background: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='4'%3E%3Cpath d='M4 0h6L7 4'/%3E%3C/svg%3E") no-repeat right center / 1em;
```

I would like to make 2 adjustments. This is the first adjustment I would like to make, in order to keep as much styling in the least specific selector:

```suggestion
  background: no-repeat right center / 1em;
```